config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -6,14 +6,14 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "true"
-    containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:7.11.0
+    containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:7.12.0
     description: RHPAM Kogito Operator for deployment and management of Kogito services.
     operators.operatorframework.io/builder: operator-sdk-v1.2.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
     repository: https://github.com/kiegroup/rhpam-kogito-operator
     support: Red Hat
     tectonic-visibility: ocs
-  name: rhpam-kogito-operator.v7.11.0
+  name: rhpam-kogito-operator.v7.12.0-1
```

Comment:
Why we have `0-1` after its name?

Reply:
it is required by prod builds if I am not mistaken.
@AntStephenson probably can give us more details.

Reply:
@spolti actually, I am not sure about this, I guess it needs to match the NVR from Brew of the Operator image being built before it? can maybe Tommy confirm it? if so it would have to be bumped to -3 as we already built this image a few times already (though they are invalid).

Reply:
For example, see here: https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=1789115 .. when we do the build after these changes are merged this will be replaced with rhpam-7-kogito-rhel8-operator-container-7.12.0-3

Reply:
Yeah so I guess it does reference the release version from brew, in that case this should be rhpam-kogito-operator.v7.12.0-3 but as I wrote on GChat I can manually change it locally and raise a PR for whatever the final change is if you prefer. Otherwise you can change it now in this PR.